runner/debug/vault.go

```diff
@@ -0,0 +1,123 @@
+package debug
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/hcdiag/op"
+	"github.com/hashicorp/hcdiag/redact"
+	"github.com/hashicorp/hcdiag/runner"
+)
+
+var _ runner.Runner = VaultDebug{}
+
+// VaultDebugConfig is a config struct for VaultDebug runners
+type VaultDebugConfig struct {
+	Compress        string
+	Duration        string
+	Interval        string
+	LogFormat       string
+	MetricsInterval string
+	Targets         []string
+	Redactions      []*redact.Redact
+}
+
+// VaultDebug represents a VaultDebug runner
+type VaultDebug struct {
+	Compress        string           `json:"compress"`
+	Duration        string           `json:"duration"`
+	Interval        string           `json:"interval"`
+	LogFormat       string           `json:"logformat"`
+	MetricsInterval string           `json:"metricsinterval"`
```

Comment:
For consistency with other runners, I think snake_case may be more appropriate for the JSON tags:
```suggestion
	LogFormat       string           `json:"log_format"`
	MetricsInterval string           `json:"metrics_interval"`
```

Reply:
Great catch -- done!